Ginger/Ginger/AutomatePageLib/NewAutomatePage.xaml.cs

```diff
@@ -738,19 +738,19 @@ private void App_AutomateBusinessFlowEvent(AutomateEventArgs args)
                     UpdateAutomatePageRunner();
                     break;
                 case AutomateEventArgs.eEventType.RunCurrentAction:
-                    RunAutomatePageAction((Tuple<Activity,Act>)args.Object,  false);
+                    await RunAutomatePageAction((Tuple<Activity, Act>)args.Object, false).ConfigureAwait(false);
                     break;
                 case AutomateEventArgs.eEventType.RunCurrentActionAndMoveOn:
-                    RunAutomatePageAction((Tuple<Activity, Act>)args.Object, false, true);
+                    await RunAutomatePageAction((Tuple<Activity, Act>)args.Object, true).ConfigureAwait(false);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this default value assigned to parameter 'moveToNextAction'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4817688)